docs/test-data.md

```diff
@@ -87,5 +88,8 @@ The table below details the available test data for each Self-Service API:
 | [City Search](https://developers.amadeus.com/self-service/category/trip/api-doc/city-search){:target="\_blank"}  | No data restrictions in test. |
 
 
+## Video Tutorial
 
+Check out this video "Test vs Production" from [Get Started series](https://youtube.com/playlist?list=PLBehidtj-OiqQ0sIHBPvwf-8GAjMTJehF). 
```

Comment:
Same here, I would write it as:
You can check this [video](https://www.youtube.com/watch?v=UU1EaWEpb5U) to know more about the differences between test and production from the [Get Started with Amadeus APIs](https://youtube.com/playlist?list=PLBehidtj-OiqQ0sIHBPvwf-8GAjMTJehF) series.